errors/error.go

```diff
@@ -73,6 +75,8 @@ type NCError struct {
 	// Contains stack trace from the initial place when the error
 	// was raised.
 	Stack []string
+	// Raw stack trace in the form of program counters, as returned by the runtime.Callers
+	RawStack *stack
```

Comment:
Could be embedded instead? Dunno if it would work with the type being unexported :thinking:

Reply:
Not sure what would not work here, but if there is no clear reason for embedding then I'd leave it as it is

Reply:
Rationale would be no need for the `StackTrace` method